src/main/resources/js/lgtm-addon.js

```diff
@@ -67,54 +65,6 @@ function loadProjects() {
 	});
 }
 
-function handleProjectChange(event) {
```

Comment:
Remove the call to this function in loadProjects() `AJS.$(projectFieldId).on('change', handleProjectChange);`

Reply:
Oops. 👍